widget-missing-in-kendoui-professional-package.md

```diff
@@ -0,0 +1,97 @@
+
+---
+title: Widgets from the Kendo UI Core/Professional Package Don't Load
+description: After installing Kendo UI via the Package Manager, widget XXXX doesn't work
+type: Troubleshooting
+page_title: Widgets from the Kendo UI Core/Professional Package Don't Load
+slug: widget-missing-in-kendoui-professional-package
+position: 0
+tags: kendo, package manager, widget, error
+ticketid: 1017633
+
+---
+
+## Environment 
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code</td>
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Package Manager</td>
+  </tr>
+  <tr>
+    <td>Mobile development type</td>
+    <td>Hybrid (Cordova app)</td>
+  </tr>
+  <tr>
+    <td>Code (AppBuilder) client</td>
+    <td>All</td>
+  </tr>
+</table>
+
+## Description
+After installing Kendo UI Core or Professional via the Manage Packages window, widgets from the respective bundle (e.g. Grid, Scheduler, DropDownList) still don't load. 
+
+If the widgets are instantiated in code, an error *Uncaught TypeError: $(...).kendo[WidgetName] is not a function* is thrown.
+
+## Cause
+
+The Kendo UI Core and Professional packages in the Package Manager are do not include the entire Core and Professional libraries. These packages are tailored for mobile development only and include the following:
+
+* **Kendo UI Core**: common Kendo UI framework features + Kendo UI Mobile
+* **Kendo UI Professional**: common Kendo UI framework features + Kendo UI Mobile + Kendo UI DataViz
+
+Kendo UI Web widgets are not included in these packages, as most of them are not optimized for mobile app development.
+
+## Solution
+
+There isn't a way to automatically add the full Kendo UI Core/Professional package in your app in this scenario. However, you can do this manually, as described in the next section.
+
+## Workaround
+
+You can manually add the full Kendo UI libraries to your project and reference them. 
+
+To do this, follow the steps below for the library of your choice:
+
+* Download and reference **Kendo UI Core**
+	1. Download the Kendo UI Core archive from here: <a href="http://www.telerik.com/download/kendo-ui-core" title="Kendo UI Core" target="_blank">Kendo UI Core</a>. 
+	2. From the **js** and **styles** folders of the Kendo UI archive copy the following files to your project:
+		* *kendo.ui.core.min.js*
+		* *kendo.common.min.css*
+		* *kendo.[skinName].min.css*, e.g. *kendo.flat.min.css*
+		* *kendo.mobile.all.min.css*
+	Also, copy from the styles folder the [skinName] folder, e.g. Flat into your project's kendo/styles folder.
+	3. Add references to the files in your app's index.html file.
+	4. 
+	```&lt;link rel="stylesheet" href="styles/kendo.common.min.css" /&gt;
```

Comment:
Move the opening and closing code block tags to separate lines. This will make the use of lt and gt for escaping the triangle brackets unnecessary.